schema-registry/serdes/src/main/java/com/hortonworks/registries/schemaregistry/serdes/avro/MessageContext.java

```diff
@@ -15,20 +15,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.hortonworks.registries.schemaregistry.avro.serdes;
-
-import java.io.InputStream;
-import java.util.Map;
+package com.hortonworks.registries.schemaregistry.serdes.avro;
```

Comment:
Create new class for this instead of moving this class from test to src and change fields etc, same for serializer and deserializer too.

Reply:
Fixed.